src/services/housing.js

```diff
@@ -30,6 +26,13 @@ import './user';
  * @property {String} OffCampusProgram The name of department of this applicant's off-campus program, or 'None'
  */
 
+/**
+ * @global
+ * @typedef ApplicantPartialInfo
+ * @property {String} Username The username of this applicant
+ * @property {String} OffCampusProgram The name of department of this applicant's off-campus program, or 'None'
+ */
+
 /**
  * Note: Properties 'HallRank' and 'HallName' must be capitalized to match the backend
  * @global
```

Comment:
Why is this Note written twice?

Reply:
probably a weird 'git merge', because I think I moved it from the top to the bottom or vice-versa. I will fix it rn